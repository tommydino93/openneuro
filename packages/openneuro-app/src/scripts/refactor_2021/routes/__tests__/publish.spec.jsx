import React from 'react'
import { shallow } from 'enzyme'
import Publish from '../publish.jsx'

describe('Publish dataset route', () => {
  it('renders with common props', () => {
    const wrapper = shallow(<Publish datasetId="ds00001" />)
    expect(wrapper).toMatchSnapshot()
  })
})
