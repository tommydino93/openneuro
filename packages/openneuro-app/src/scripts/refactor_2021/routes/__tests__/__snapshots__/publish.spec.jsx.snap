// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Publish dataset route renders with common props 1`] = `
<Redirect
  to={
    Object {
      "pathname": "/datasets/ds00001/metadata",
      "state": Object {
        "submitPath": "/datasets/ds00001/publish",
      },
    }
  }
/>
`;
